public/app/types/unified-alerting-dto.ts

```diff
@@ -56,11 +56,35 @@ export enum PromApplication {
   Prometheus = 'Prometheus',
 }
 
+export interface PromBuilderInfoSeriesMetric {
+  metric: {
+    branch: string;
+    goversion: string;
+    instance: string;
+    job: string;
+    revision: string;
+    version: string;
+  };
+}
+export interface PromBuildInfoSeriesResponse {
+  status: string;
+  data: {
+    resultType: string;
+    result: PromBuilderInfoSeriesMetric[];
+  };
+}
+
+export type PrometheusVersionThroughMimir = 'unknown';
+/**
+ * Describes the result of the buildinfo API endpoint, Mimir will not return a prometheus version, and instead a string of 'unknown'
```

Comment:
This isn't true (in most cases), this metadata is added to mimir during build, and some of the local development environments don't have this configured out of the box. Update this comment please!